cloud-discord/cloud-jda/src/main/java/cloud/commandframework/jda/JDACommandListener.java

```diff
@@ -126,7 +127,7 @@ public final void onMessageReceived(final @NonNull MessageReceivedEvent event) {
     }
 
     private void sendMessage(final @NonNull MessageReceivedEvent event, final @NonNull String message) {
-        event.getChannel().sendMessage(message).queue();
+        event.getChannel().sendMessage(message).complete();
```

Comment:
Why would you change this to complete? Complete blocks the thread until the operation is complete which can be problematic when executed asynchronously.

Reply:
You could also use [submit()](https://github.com/DV8FromTheWorld/JDA/wiki/7%29-Using-RestAction#using-submit) so it's up to the user to decide whether the operation should be blocking or not.

Reply:
I changed this to complete because i was debugging something must've forgot. Submit is cool but it wouldn't work for this, the result is contained in this when complete. If people want to complete their rest actions themselves they should add their own exception handler